.travis.yml

```diff
@@ -1,15 +1,24 @@
 language: clojure
 lein: lein2
-install: (cd cloverage && lein2 deps) && (cd lein-cloverage && lein2 deps)
-script: (cd cloverage && lein2 test) && (cd lein-cloverage && lein2 test)
-sudo: false
-cache:
-  directories:
-    - $HOME/.m2
+sudo: true
+
+before_install: yes | sudo lein upgrade
```

Comment:
Sorry, I wasn't clear. You can specify: `lein: 2.7.1` instead of `lein: lein2` and then you don't need sudo.